src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/jpa/entity/LicenceConditionChangeAuditEntity.kt

```diff
@@ -41,6 +41,9 @@ data class LicenceConditionChangeAuditEntity(
 
   @Column(name = "seen")
   val seen: Boolean = false,
+
+  @Column(name = "confirmation_date")
+  val confirmationDate: LocalDateTime?,
```

Comment:
default to null maybe, as we'd never want to initially construct this with a value